config/samples/job_v1alpha1_snoopyjob.yaml

```diff
@@ -10,5 +10,5 @@ spec:
     }
   targetNamespace: cnf-telco
   timer: "30s"
-  dataServiceIP: "172.30.30.60"
+  dataServiceIP: "172.30.61.241"
```

Comment:
Im concerned with the process that results in this file changing as individual dev environments change.  Might be best to just set this to a slug,  and then set local .gitignore files to not see the changes.  Otherwise this file will end up getting changed in PRs that have nothing to do with it.